emission/analysis/classification/inference/mode.py

```diff
@@ -149,13 +226,16 @@ def generateFeatureMatrixAndResultVectorStep(self):
 
       # So we limit the records to the size of the matrix that we have created
       for (i, section) in enumerate(self.confirmedSections.limit(featureMatrix.shape[0]).batch_size(300)):
-        try:
-            self.updateFeatureMatrixRowWithSection(featureMatrix, i, section)
-            resultVector[i] = self.getGroundTruthMode(section)
-            if i % 100 == 0:
-                logging.debug("Processing record %s " % i)
-        except Exception, e:
-            logging.debug("skipping section %s due to error %s " % (section, e))
+       try:
+        self.updateFeatureMatrixRowWithSection(featureMatrix, i, section)
+        resultVector[i] = self.getGroundTruthMode(section)
+    
+
+        if i % 100 == 0:
+            logging.debug("Processing record %s " % i)
+      except Exception, e:
+        logging.debug("skipping section %s due to error %s " % (section, e))
```

Comment:
There isn't a difference. I had some print statements around there and commented it some of it out and then put it back in. But it is the same code.